include/users/modules/class-bulk-delete-users-by-user-role.php

```diff
@@ -103,8 +103,14 @@ public function render() {
 	 */
 	public function process() {
 		$delete_options                   = array();
+
+		/*
+		 * Nonce verification is handled in `request_handler()`.
+		 */
+		// @codingStandardsIgnoreStart
```

Comment:
You can also use single line ignore comment.

Reply:
Fixed.